server/src/controller/userController.js

```diff
@@ -41,4 +44,44 @@ const userSignUp = async (req, res) => {
   }
 };
 
-export { userLogin, userSignUp };
+// Get user data
+const getMe = async (req, res) => {
+  try {
+    const { _id, name, email, postalCode, roles } = await User.findById(
+      req.user.id
+    );
+
+    res.status(200).json({
+      id: _id,
+      name,
+      email,
+      postalCode,
+      roles,
+    });
+  } catch (err) {
+    res.status(400).json({ error: err.message });
+  }
+};
+
+// Update user data
+const updateMe = async (req, res) => {
+  try {
+    const { _id, name, email, postalCode, roles } = await User.findOneAndUpdate(
+      { _id: req.user.id },
+      { ...req.body },
+      { new: true }
+    );
+
+    res.status(200).json({
+      id: _id,
+      name,
+      email,
+      postalCode,
+      roles,
+    });
+  } catch (err) {
+    res.status(400).json({ error: err.message });
+  }
+};
+
+export { userLogin, userSignUp, getMe, updateMe };
```

Comment:
Nice work! Would it make sense to include `deleteMe` as well?

Reply:
That's a great idea! I'll add that now.